config/default/system.advisories.yml

```diff
@@ -0,0 +1,2 @@
+interval_hours: 6
+enabled: true
```

Comment:
We may want to turn this off. https://www.drupal.org/node/3206473

Reply:
they have really messed with our demos as of late. "don't look at that giant security warning, instead look at this new button you can press."

Reply:
We have a lot of people getting notified through other means. We don't hundreds of sites making outbound requests on cron.